src/StoreKeeper/WooCommerce/B2C/Tools/TaskHandler.php

```diff
@@ -590,6 +590,9 @@ private function reportFailedTask($task_id, $exception)
             $data = TaskModel::get($task_id);
             if ($data) {
                 $metaData['failed-task-try'] = $data['times_ran'];
+                $generatedMetadata = LoggerFactory::generateMetadata('task-in-queue-failed', $exception, $metaData);
```

Comment:
`metaData` should come from $data

Reply:
Changed `$metaData` to `$data['meta_data']`.